src/api/routes/commentReactionsRoute.js

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import commentReaction from '../controllers/commentReactionsController';
+import checkValidToken from '../../middlewares/checkValidToken';
+import errorHandler from '../../middlewares/errorHandler';
+import likeDislikeCheck from '../../middlewares/commentReaction';
+
+const likeDislikeRouter = new Router();
+likeDislikeRouter.post('/like/:commentId',
```

Comment:
@murediane like and dislike in these routes should be `likes` and `dislikes` i.e plural